draft-ietf-rtcweb-jsep.xml

```diff
@@ -1449,7 +1457,8 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
           and continues until setRemoteDescription is called with
           either a final answer, at which point the PeerConnection can
           fully adopt the pending local description, or a rollback,
-          which results in a revert to the current local
+          which causes the JSEP implementation to revert back from the
```

Comment:
I considered this implicit from the surrounding text, i.e., the previous clause refers to the pending description, and this clause is positioned as an alternative. IOW, seems a bit redundant to spell this out, although I don't feel super strongly.